appcenter/AppCenter.d.ts

```diff
@@ -37,3 +37,9 @@ export function setEnabled(enabled: boolean): Promise<void>;
 export function setUserId(userId: string | null): Promise<void>;
 export function getInstallId(): Promise<string>;
 export function setCustomProperties(properties: CustomProperties);
+export function setAuthToken(authToken: string | null): void;
+export function setAuthTokenListener(listener: AuthTokenListener) : void;
+
+export interface AuthTokenListener {
+    acquireAuthToken?: () => string;
```

Comment:
is the callback optional? or did you mean you can return string | null

Reply:
or both maybe

Reply:
Good catch. Fixed now.